internal/command/deploy/deploy.go

```diff
@@ -155,13 +155,26 @@ func DeployWithConfig(ctx context.Context, appConfig *appconfig.Config, args Dep
 	case err != nil:
 		return err
 	case isV2App:
-		if err := appConfig.EnsureV2Config(); err != nil {
+		if err = appConfig.EnsureV2Config(); err != nil {
```

Comment:
no need to remove the `:`, for simple function calls like this it is preferred to stick to this style:

```go
if err := functionCall(); err != nil {
  return err
}
```